app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -152,10 +178,10 @@ function AddressSearch({
     validatedFieldRef,
   } = useUspsLocations();
 
-  const onTextInputChange = useCallback((event: React.ChangeEvent<HTMLInputElement>) => {
+  const onTextInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
The previous code would reduce the likelihood of re-evaluation of the `<TextInput>` component, so I'd be inclined to leave this unless it was causing problems. That said, the gain is probably marginal so either way is ok.